mteb/tasks/Clustering/nob/VGHierarchicalClustering.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
+from mteb.abstasks import TaskMetadata
+from mteb.abstasks.AbsTaskClusteringFast import AbsTaskClusteringFast
+
+
+def split_labels(record: dict) -> dict:
+    record["labels"] = record["labels"].split(",")[:2]
+    return record
+
+
+class VGHierarchicalClusteringP2P(AbsTaskClusteringFast):
```

Comment:
Should we outdate VG?

Reply:
I think I added superseeded_by to the OG task